### PR TITLE
Portfolio page

### PR DESCRIPTION
**Pull request by HeatherHerries**
I created the portfolio page without the .map() function, but i did comment out the code that I created for that  in both the PortfolioPage.js and the PortCardHolder.js. Take a look and let me know what you think. Thanks bro!

### CODE REVIEW

File: src/Components/PortCardHolder.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import '../Styles/PortCardHolder.css'
+import Subheading from './Subheading'
+
+function PortCardHolder({sbText, pchPara, pchImg1, pchImg2, pchImg3, pchImg4, pchImg5, pchImg6}) {
```
Comment (zcherries) [inline src/Components/PortCardHolder.js]:
```suggestion
function PortCardHolder({sbText, pchPara, pchImgs}) {
```

File: src/Components/PortCardHolder.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      <div className="pch-imgs">
+        <img className='img1'src={pchImg1} alt="" />
+        <img className='img2'src={pchImg2} alt="" />
+        <img className='img3'src={pchImg3} alt="" />
+        <img className='img4'src={pchImg4} alt="" />
+        <img className='img5'src={pchImg5} alt="" />
+        <img className='img6'src={pchImg6} alt="" />
+        {/* {
+          pchImgs.map(pchImg => (<img src={pchImg} alt=""/>))
+        }  */}
```
Comment (zcherries) [inline src/Components/PortCardHolder.js]:
```suggestion
        {pchImgs.map((pchImg, index) => <img className={`img${index + 1}`} key={index} src={pchImg} alt="" />)}
```

File: src/Pages/PortfolioPage.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+          pchImg5={DevImg5}
+          pchImg6={DevImg6}
+          // pchImgs={[
+          //   {DevImg1}, 
+          //   {DevImg2}, 
+          //   {DevImg3}, 
+          //   {DevImg4},
+          //   {DevImg5}, 
+          //   {DevImg6} 
+          // ]}
```
Comment (zcherries) [inline src/Pages/PortfolioPage.js]:
```suggestion
          pchImgs={[
            {DevImg1}, 
            {DevImg2}, 
            {DevImg3}, 
            {DevImg4},
            {DevImg5}, 
            {DevImg6} 
          ]}
```

File: src/Pages/PortfolioPage.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+          <PortCardHolder
+            sbText="Design" 
+            pchPara="My thoughtful visual designs will celebrate your brand, connect with your audience, and create a natural flow to navigate the site. I hope these samples of my work will spark your imagination.
+            "
+            pchImg1={DesignImg1}
+            pchImg2={DesignImg2}
+            pchImg3={DesignImg3}
+            pchImg4={DesignImg4}
+            pchImg5={DesignImg5}
+            pchImg6={DesignImg6}
```
Comment (zcherries) [inline src/Pages/PortfolioPage.js]:
```suggestion
            pchImgs={[
              DesignImg1,
              DesignImg2,
              DesignImg3,
              DesignImg4,
              DesignImg5,
              DesignImg6,
              ]}
```

File: src/Components/PortCardHolder.js (hunk 2 above)
Comment (HeatherHerries) [reply src/Components/PortCardHolder.js]: After plugging in all the requested changes I did not get any errors, but this is what it rendered:

Screen Shot 2022-11-20 at 9 28 13 AM

**Pull request merged**